sdk/core/Azure.Core.Experimental/src/DynamicData/DynamicData.Converters.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.Core.Dynamic
+{
+    public partial class DynamicData
+    {
+        // From: https://github.com/Azure/autorest.csharp/blob/d835b0b7bffae08c1037ccc5824e928eaac55b96/src/assets/Generator.Shared/TypeFormatters.cs#L14
```

Comment:
All this copying motivates the need to [move these conversion operations into Azure.Core](https://github.com/Azure/azure-sdk-for-net/issues/32006) to ensure consistency across usage.